cdc/server_test.go

```diff
@@ -19,6 +19,9 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/pingcap/ticdc/cdc/kv"
+	"go.etcd.io/etcd/clientv3"
+
 	"github.com/pingcap/check"
 	"github.com/pingcap/ticdc/pkg/config"
 	"github.com/pingcap/ticdc/pkg/etcd"
```

Comment:
Better to remove the blank line and group the import

Reply:
done
thx ~!